packages/front-end/components/Features/VariationsInput.tsx

```diff
@@ -156,196 +176,47 @@ export default function VariationsInput({
-                            {i < variations.length - 1 && (
-                              <button
-                                className="dropdown-item"
-                                onClick={(e) => {
-                                  e.preventDefault();
-
-                                  const newValues = [...variations];
-                                  [newValues[i], newValues[i + 1]] = [
-                                    newValues[i + 1],
-                                    newValues[i],
-                                  ];
+            <DndContext
+              sensors={sensors}
+              collisionDetection={closestCenter}
+              onDragEnd={({ active, over }) => {
+                if (active.id !== over.id) {
+                  const oldIndex = getVariationIndex(active.id);
+                  const newIndex = getVariationIndex(over.id);
 
-                                  setVariations(newValues);
-                                }}
-                              >
-                                move down
-                              </button>
-                            )}
-                            {variations.length > 2 && (
-                              <button
-                                className="dropdown-item text-danger"
-                                onClick={(e) => {
-                                  e.preventDefault();
+                  if (oldIndex === -1 || newIndex === -1) return;
 
-                                  const newValues = [...variations];
-                                  newValues.splice(i, 1);
-
-                                  const newWeights = distributeWeights(
-                                    newValues.map((v) => v.weight),
-                                    customSplit
-                                  );
-
-                                  newValues.forEach((v, j) => {
-                                    v.weight = newWeights[j] || 0;
-                                  });
-                                  setVariations(newValues);
-                                }}
-                                type="button"
-                              >
-                                remove
-                              </button>
-                            )}
-                          </MoreMenu>
-                        </div>
-                      )}
-                    </div>
-                  </td>
-                </tr>
-              );
-            })}
+                  const newVariations = arrayMove(items, oldIndex, newIndex);
 
+                  setVariations(newVariations);
+                }
+              }}
+            >
+              <SortableContext
+                items={items}
+                strategy={verticalListSortingStrategy}
+              >
+                {items.map(({ ...variation }, i) => (
+                  <SortableVariationRow
+                    i={i}
+                    id={variation.id}
```

Comment:
We aren't actually using this `id` prop in `Variations` since we grab the id from `props.variation.id` (and not `props.id`); Can delete it